src/clr-icons/clr-icons-element.ts

```diff
@@ -135,7 +137,7 @@ ClarityIconElement.prototype.attributeChangedCallback = function(
   // This means even if the shape is not found, the injected shape will have the user-given size.
 
   if (attributeName === 'shape') {
-    this.currentShapeAttrVal = newValue.split(/\s/)[0];
+    this.currentShapeAttrVal = newValue.split(/\s/)[0].toLowerCase();
```

Comment:
Is this complex enough or being used enough to abstract it into a function?

That way you could just test the function in isolation and not have two different unit tests testing the same thing in `clr-icons.spec.ts`.

Reply:
I manage to test it with one test only. 

And moved it into function `_normalizeShape` so feature changes will be mostly oneliners ..